Cesium3DTilesSelection/include/Cesium3DTilesSelection/IPrepareRendererResources.h

```diff
@@ -46,7 +50,8 @@ struct TileLoadResultAndRenderResources {
  * {@link TilesetExternals} structure that can be obtained
  * via {@link Tileset::getExternals}.
  */
-class CESIUM3DTILESSELECTION_API IPrepareRendererResources {
+class CESIUM3DTILESSELECTION_API IPrepareRendererResources
+    : public CesiumRasterOverlays::IPrepareRasterOverlayRendererResources {
```

Comment:
Certainly inheritance should model IS-A. In this case, we're saying "a thing that can create renderer resources for a tileset _IS A_ thing that can create renderer resources for a raster overlay." Which is true enough. And composition doesn't really work for an interface like this, because an interface can't "contain" anything. `IPrepareRendererResources` could have a method that gets `IPrepareRasterOverlayRendererResources` instead of inheriting from it, which would at least allow implementors to compose the implementation of one inside the other. But then that either breaks backward compatibility (by removing the `prepareRasterInLoadThread` method and friends from `IPrepareRendererResources`), or it ends up with `IPrepareRendererResources` being a `IPrepareRasterOverlayRendererResources` anyway (by leaving `prepareRasterInLoadThread` and friends, which also exist on this interface). In the latter case, we end up modeling IS-A anyway, so why make our lives harder by fighting it?